### ISSUE TITLE
Add Support for "Audio" Device Type

### ISSUE THREAD

**Issue by mgrosman** [prasad-edlabadka/homebridge-tuya-ir#14]
Thanks for creating this plugin!  Would it be possible to add support for Audio device group?

**Comment by prasad-edlabadka** [comment]
What would audio device map to? There isn't a "Audio device" in HomeKit. Airplay 2 is not supported by home bridge.

**Comment by mgrosman** [comment]
Maybe it can be mapped to "uncategorized"? Or even as a switch if that would enable to control on/off?
For instance in the Google home app the audio device is recognized as audio but cannot be controlled for any functions other on/off

**Comment by bkarlson** [comment]
Yes, even controlling some device as a switch would be helpful, this way we can activate amplifiers, TVs etc.
